src/lib/account/LoginForm.svelte

```diff
@@ -42,22 +44,35 @@
 </script>
 
 <form on:submit|preventDefault={login} action="/account/inloggen.json" method="POST">
-	<label for="username">Gebruikersnaam</label>
-	<input bind:value={username} name="username" type="text" disabled={offline} />
-	<label for="password">Wachtwoord</label>
-	<input bind:value={password} name="password" type="text" disabled={offline} />
-	<button type="submit">Inloggen</button>
+	<label for="username">Gebruikersnaam:</label>
+	<input bind:value={username} name="username" type="text" disabled={offline} autofocus />
+	<label for="password">Wachtwoord:</label>
+	<input bind:value={password} name="password" type="password" disabled={offline} />
+	<SubmitButton>Inloggen</SubmitButton>
 </form>
 
 <style>
 	form {
 		display: flex;
 		flex-flow: column nowrap;
-		max-width: 80%;
 		margin: 0 auto;
+		height: 50vh;
```

Comment:
Deze hoogte is per device verschillend. Ik denk dat het handiger is als we hem altijd 100% van het scherm laten vullen. Hoe denkt jij hier over?

Reply:
Ik heb dit getest op een groot scherm dan wordt hij halverwegen afgekapt

Reply:
Je hebt gelijk. Het ziet er niet echt uit. Als ik 100% gebruik dan komt hij er ook raar op. Wellicht gooi ik die achtergrond gewoon hier weg! Ik ga nog even kijken naar een oplossing